src/Components/GEOSldas_GridComp/.gitignore

```diff
@@ -1 +1,2 @@
 /@GEOSgcm_GridComp
+/@FVdycoreCubed_GridComp
```

Comment:
Good eye catching this. I guess I never did a `git status` after making my changes!